test/integration/infrastructure/infrastructure_test.go

```diff
@@ -0,0 +1,613 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package infrastructure_test
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	gcpinstall "github.com/gardener/gardener-extension-provider-gcp/pkg/apis/gcp/install"
+	gcpv1alpha1 "github.com/gardener/gardener-extension-provider-gcp/pkg/apis/gcp/v1alpha1"
+	"github.com/gardener/gardener-extension-provider-gcp/pkg/controller/infrastructure"
+	"github.com/gardener/gardener-extension-provider-gcp/pkg/gcp"
+	. "github.com/gardener/gardener-extension-provider-gcp/test/integration/infrastructure"
+
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	"github.com/gardener/gardener/pkg/operation/common"
+	gardenerutils "github.com/gardener/gardener/pkg/utils"
```

Comment:
In my last PRs I have put everything from `github.com/gardener` in one group. Maybe my understanding is wrong how we are importing packages?

Reply:
I think the convention is to have the second import group with imports that are from the same project (not from the same organisation). If my understanding is wrong, please speak up.

Reply:
See https://github.com/gardener/gardener/pull/2655